ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -118,8 +118,10 @@ ${listenerText}
 sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" ;\\
 sudo apt-get update && sudo apt-get install boundary ;\\
 boundary server -config="${configPath}/pki-worker.hcl"`;
-    const hcpContent = `wget -q https://releases.hashicorp.com/boundary-worker/0.12.0+hcp/boundary-worker_0.12.0+hcp_linux_amd64.zip ;\\
-sudo apt-get update && sudo apt-get install unzip ;\\
+
+    const hcpContent = `sudo apt-get update && sudo apt-get install jq unzip ;\\
+wget -q "$(curl -fsSL "https://api.releases.hashicorp.com/v1/releases/boundary-worker/latest?license_class=hcp" \
+| jq -r '.builds[] | select(.arch == "amd64" and .os == "linux") | .url')" ;\\
```

Comment:
Non-blocking question: is the reason for having `sudo apt-get install jq unzip` on line 122 because `jq` and `unzip` might not be already installed in the user's linux system? If so, should `wget` also be added to `sudo apt-get install jq unzip`, or is that not necessary ?

Reply:
We could, I thought about it but `wget` is normally installed with most linux distros IIRC. `jq` and `unzip` aren't so they definitely to be installed.